scripts/contractInteraction/liquidity_miningV1.py

```diff
@@ -0,0 +1,92 @@
+from brownie import *
```

Comment:
I'm not quite sure why do we need this functions? Aren't all this configs taken from the storage of `V0` ?